internal/capability/auditor.go

```diff
@@ -10,30 +10,29 @@ import (
 // It has methods to create a workqueue with all the package and audit requirements for a
 // particular audit run
 type Auditor interface {
-	BuildWorkQueueByCatalog(opts operator.OperatorCheckOptions) error
 	RunAudits() error
 }
 
 // capAuditor implements Auditor
-type capAuditor struct {
+type CapAuditor struct {
 
-	// Workqueue holds capAudits in a buffered channel in order to execute them
-	WorkQueue chan CapAudit
-}
+	// AuditPlan holds the tests that should be run during an audit
+	AuditPlan []string
 
-// BuildAuditorByCatalog creates a new Auditor with workqueue based on a selected catalog
-func BuildAuditorByCatalog(options operator.OperatorCheckOptions) (capAuditor, error) {
+	// CatalogSource may be built-in OLM or custom
+	CatalogSource string
+	// CatalogSourceNamespace will be openshift-marketplace or custom
+	CatalogSourceNamespace string
 
-	var auditor capAuditor
-	err := auditor.BuildWorkQueueByCatalog(options)
-	if err != nil {
-		logger.Fatalf("Unable to build workqueue err := %s", err.Error())
-	}
-	return auditor, nil
+	// FilterPackages is a subset of packages to be tested from a catalogSource
+	FilterPackages []string
```

Comment:
I totally agree. But proposed using packages instead of package-list since we already have list-packages which can lead to confusion. Just pushed the change.